Calibration/EcalCalibAlgos/test/ecalPedestalPCL_step2_cfg.py

```diff
@@ -32,7 +32,8 @@
 
 # Other statements
 from Configuration.AlCa.GlobalTag import GlobalTag
-process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:run2_data', '')  
+#process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:run2_data', '')  
+process.GlobalTag = GlobalTag(process.GlobalTag, '92X_dataRun2_Express_v2_withECALPedHLT', '') 
 
```

Comment:
hello @argiro 
please don't hard code this GT which we provided you for your local initial tests
run2_data does provide you with the record/label you need, see : 
https://cms-conddb-int.cern.ch/cmsDbBrowser/diff/Prod/gts/92X_dataRun2_v2/92X_dataRun2_v5